Tools/TestWebKitAPI/Tests/WebKitCocoa/PasteHTML.mm

```diff
@@ -88,7 +88,12 @@ void writeHTMLToPasteboard(NSString *html)
     EXPECT_WK_STREQ("hello, world", [webView stringByEvaluatingJavaScript:@"editor.textContent"]);
 }
 
+// rdar://138144869
+#if PLATFORM(IOS)
```

Comment:
Given this test is only flaky on debug, can we change this to `#if PLATFORM(IOS) && !defined(NDEBUG)`?
Otherwise, we're gonna lose test coverage on iOS.